src/hooks/ykfde

```diff
@@ -150,6 +150,7 @@ ykfde_do_it() {
 
   if [ "$_rc" -eq 0 ]; then
     message "   Decryption was successful."
+    touch /.done
```

Comment:
can you make a comment above describing what's of this? i.e. 
```
# encryptssh support
touch /.done
```

Reply:
Done.